src/index.js

```diff
@@ -1,5 +1,20 @@
 import './sass/main.scss';
 import './partials-js/chart';
-import {getCityWeather} from './partials-js/data_fetch';
+import {getCityWeather, citiesData} from './partials-js/data_fetch';
+import axios from 'axios';
 
-console.log(getCityWeather("london"));
+let defaultCity = [];
+
+document.addEventListener("DOMContentLoaded", async () => {
+    await axios.get("https://api.openweathermap.org/data/2.5/forecast?lat=52.2319581&lon=21.0067249&units=metric&appid=86882c431a5c1fa03f48939e3b313043")
+    .then((response) => {
+        defaultCity.push(response);
+    })
+    .catch((error) => {
+        return error;
+    });
+});
+
```

Comment:
zamiast pisać całej funkcjonalności od nowa użyj juz kodu co napisałeś, w addeventlistener  użuj funkcji getCityWeather z domyślnym miastem dla na warszasa

Reply:
No właśnie tak nie mogę zrobić, bo ta funkcja jako parametr przyjmuje event, a nie stringa. W takim przypadku funkcja zwróci błąd, bo funkcja getCoordinates potrzebuje eventu żeby dostać się do ev.target.value.

Reply:
ewentualnie mogę jeszcze spróbować jakoś zrefraktoryzować tą funkcję, żeby mogła przyjąć też stringa, ale nie wiem czy jest to potrzebne bo i tak za bardzo to nie wpłynie na optymalizacje